python/dataproc_templates/util/template_constants.py

```diff
@@ -16,10 +16,24 @@
 PROJECT_ID_PROP = "project.id"
 
 # Data
+TEXT_HEADER = "header"
+TEXT_INFER_SCHEMA = "inferSchema"
+INPUT_DELIMITER = "delimiter"
+INPUT_COMPRESSION = "compression"
+COMPRESSION_BZIP2 = "bzip2"
+COMPRESSION_GZIP = "gzip"
+COMPRESSION_DEFLATE = "deflate"
+COMPRESSION_LZ4 = "lz4"
+COMPRESSION_NONE = "None"
+DELIMITER_COMMA = ","
+DELIMITER_PIPE = "|"
+DELIMITER_SLASH = "/"
+DELIMITER_SEMICOLON =";"
```

Comment:
Do we need this?

Reply:
Not required. Removed the unused constants.